src/App.tsx

```diff
@@ -1,27 +1,46 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+import { getTodos } from './api';
+
 const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
   const [selectedUserId, setSelectedUserId] = useState(0);
 
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => setTodos(todosFromServer));
+  }, []);
+
   return (
     <div className="App">
       <div className="App__sidebar">
-        <TodoList />
+        {todos.length ? (
+          <TodoList
+            todos={todos}
+            selectedUserId={selectedUserId}
+            onSelect={(userID: number) => {
+              setSelectedUserId(userID);
+            }}
```

Comment:
no need to do it here, you can create a method inside TodoList, just pass here setSelectedUserId fucntion